src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeController.cs

```diff
@@ -35,6 +35,6 @@ public async Task PostApprentice(CreateApprenticeAccountCommand command)
 
         [HttpPatch("apprentices/{id}")]
         public async Task UpdateApprentice(Guid id, JsonPatchDocument<ApprenticePatchDto> changes)
-            => await _mediator.Send(new UpdateApprenticeCommand(id, changes));
+            => await _mediator.Send(new UpdateApprenticeEmailAddressCommand(id, changes));
```

Comment:
Feels odd to be doing a patch on an entity that doesn't exist! But will obviously work

Reply:
I know what you mean.  I'm not entirely happy with that URL but haven't yet come up with an appropriate REST style API for a resource that doesn't exist.

This might suggest that this entire responsibility is in the wrong place.

Reply:
Looks like, regardless of the value of the patch operation, we are always going to invoke the UpdateApprenticeEmailAddress. 

This feels wrong, we should either be using a simpler endpoint, or if we want to use the patch method, it should only update the email command if the change is explicitly for an email change. The intention should always be clear.

Reply:
Yes, I agree. It boils down to the fact that the apprentice is no longer a concern of the API, and so trying to shoehorn a change-of-apprentice-email responsibility into the domain ends up with a strange looking API.

The obvious (once I thought of it!) solution is to make jobs responsible for publishing the `ApprenticeshipEmailAddressChangedEvent` instead - no weird API needed.